addons/misra.py

```diff
@@ -21,6 +21,14 @@
 import os
 import argparse
 import codecs
+import string
+
+from itertools import izip
```

Comment:
`izip` is no longer available in Python 3, see https://stackoverflow.com/questions/32659552/importing-izip-from-itertools-module-gives-nameerror-in-python-3-x